src/rdo.rs

```diff
@@ -412,6 +413,28 @@ fn rdo_cfl_alpha(
   best_cfl
 }
 
+fn save_mc_blk(dst: &mut [u16], src: &PlaneSlice<'_>, width: usize, height: usize) {
+  let src_stride = src.plane.cfg.stride;
+
+  for (l, s) in dst.chunks_mut(width).take(height)
+                   .zip(src.as_slice().chunks(src_stride)) {
+    for (r, v) in l.iter_mut().zip(s) {
```

Comment:
You might use [copy_from_slice](https://doc.rust-lang.org/std/primitive.slice.html#method.copy_from_slice) since they are both `&[u16]`.